src/nidcpower/metadata/attributes.py

```diff
@@ -1339,6 +1341,7 @@
         },
         'lv_property': 'Inherent IVI Attributes:Instrument Identification:Serial Number',
         'name': 'SERIAL_NUMBER',
+        'repeated_capability_type': 'instruments',
```

Comment:
There's no `instruments` repeated capability yet, right?
I'm a bit surprised this doesn't error out. I think it should so that bugs are caught.

Reply:
Oops! I was so focused on the internal metadata converter not supporting `repeated_capability_type`, I totally forgot that these changes in internal metadata are for an unreleased feature (We badly need feature toggles there).

Created #1521 to catch issues like this.